### PR TITLE
EPMRPP-38853 Launches. Step view. Modals. Link issue modal improvements

### PR DESCRIPTION
**Pull request by AmsterGet**

### CODE REVIEW

File: app/src/controllers/plugins/sagas.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import { fetchDataAction } from 'controllers/fetch';
+import {
+  FETCH_PLUGINS,
+  FETCH_GLOBAL_INTEGRATIONS,
+  NAMESPACE,
+  GLOBAL_INTEGRATIONS_NAMESPACE,
+} from './constants';
+
+function* fetchGlobalIntegrations() {
+  yield put(fetchDataAction(GLOBAL_INTEGRATIONS_NAMESPACE)(URLS.globalIntegrationsByPluginName()));
```
Comment (DiscoElevator) [inline app/src/controllers/plugins/sagas.js]: For such actions you don't need sagas. You can do it the following way:
```
export const fetchGlobalIntegrationsAction = () =>
  fetchDataAction(GLOBAL_INTEGRATIONS_NAMESPACE)(URLS.globalIntegrationsByPluginName()));
```

File: app/src/pages/inside/stepPage/modals/linkIssueModal/linkIssueModal.jsx [hunk 2]
```diff
@@ -168,6 +205,9 @@ export class LinkIssueModal extends Component {
       eventInfo: STEP_PAGE_EVENTS.CANCEL_BTN_LOAD_BUG_MODAL,
     };
 
+    console.log(this.isMultipleBtsPlugins());
```
Comment (DiscoElevator) [inline app/src/pages/inside/stepPage/modals/linkIssueModal/linkIssueModal.jsx]: Looks like you forgot to remove it

File: app/src/controllers/plugins/sagas.js (hunk 1 above)
Comment (AmsterGet) [reply app/src/controllers/plugins/sagas.js]: Oh, thanks. Fixed.

File: app/src/pages/inside/stepPage/modals/linkIssueModal/linkIssueModal.jsx (hunk 2 above)
Comment (AmsterGet) [reply app/src/pages/inside/stepPage/modals/linkIssueModal/linkIssueModal.jsx]: Fixed.

**Pull request merged**